rocket_component.py

```diff
@@ -0,0 +1,102 @@
+from atmosphere import Atmosphere
+import math
+
+
+class Rocket:
+  """
+  The rocket class will calculate the forces acting upon the rocket component
+  at a given time. Units will be metric for consistency.
+  """
+
+  def __init__(self) -> None:
+
+    # Basic Properties (Falcon 9 ex.)
+    self.altitude = 0  # m
+    self.length = 70  # m
+    self.width = 4  # m
+    self.velocity = 1000  # m/s
+
+    # Mass Properties (Falcon 9 ex.)
+    self.mass_fuel = 375000  # kg
+    self.mass_flow_rate = 1450  # kg/s
+    self.mass_structure = 25000  # kg
+
+    # Engine Properties
+    self.area_exhaust = math.pi * (self.width / 2) ** 2
+    self.pressure_exhaust = 70927.5  # N/m^2
+    self.velocity_exhaust = 3000  # m/s
+
+    # Forces
+    self.drag_force = self.calc_drag_force()  # N
```

Comment:
Might want to consider grouping all of these force property sets into an update_forces function and then just call that function here.  We'll need that type of function later too.